<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bag</class>
 <widget class="QWidget" name="bag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>447</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="pokemon_icon">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>pokemon_icon</string>
   </property>
  </widget>
  <widget class="QListWidget" name="pokemon_list">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="att_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>210</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>攻击力：</string>
   </property>
  </widget>
  <widget class="QLabel" name="prolabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>属性  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>170</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>类型  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="den_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>防御力：</string>
   </property>
  </widget>
  <widget class="QLabel" name="speed_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>速度  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="level_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>等级  ：</string>
   </property>
  </widget>
  <widget class="QLabel" name="exp_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>经验值：</string>
   </property>
  </widget>
  <widget class="QLabel" name="hp_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HP    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>130</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>精灵名字</string>
   </property>
  </widget>
  <widget class="QLabel" name="pro">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>170</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>pro</string>
   </property>
  </widget>
  <widget class="QLabel" name="type">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QLabel" name="att">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>att</string>
   </property>
  </widget>
  <widget class="QLabel" name="den">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>den</string>
   </property>
  </widget>
  <widget class="QLabel" name="spe">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>spe</string>
   </property>
  </widget>
  <widget class="QLabel" name="hp">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>hp</string>
   </property>
  </widget>
  <widget class="QLabel" name="level">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>290</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>level</string>
   </property>
  </widget>
  <widget class="QLabel" name="exp">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>290</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>exp</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
